src/platform/macos.rs

```diff
@@ -38,6 +38,33 @@ pub trait WindowExtMacOS {
     /// Sets whether or not the window has shadow.
     fn set_has_shadow(&self, has_shadow: bool);
 
+    /// Sets whether the system can automatically organize windows into tabs.
+    ///
+    /// <https://developer.apple.com/documentation/appkit/nswindow/1646657-allowsautomaticwindowtabbing>
+    fn set_allows_automatic_window_tabbing(&self, enabled: bool);
+
+    /// Returns whether the system can automatically organize windows into tabs.
+    fn allows_automatic_window_tabbing(&self) -> bool;
```

Comment:
I never noticed before but modifying the automatic tabbing behaviour is actually app-wide and not specific to one window, should it be implemented on `EventLoopWindowTargetExtMacOS` and `EventLoopBuilderExtMacOS` instead?

Reply:
Hm, yeah, I haven't thought about it. We could move it all, I guess. I just got confused because it was a static method on a window.

Reply:
Moved